panther_driver/src/panther_hardware.py

```diff
@@ -93,6 +100,26 @@ def _setup_gpio() -> None:
         GPIO.setup(DRIVER_EN, GPIO.OUT, initial=0)
         GPIO.setup(E_STOP_RESET, GPIO.IN)  # USED AS I/O
 
+    @staticmethod
+    def _shutdown_host() -> None:
+        # EXAMPLE: https://www.linode.com/docs/guides/use-paramiko-python-to-ssh-into-a-server/
+        host = "10.15.20.2"
+        username = "husarion"
```

Comment:
Const like this shouldn't be in the code. 
Since this process is running on the host, we can access username and host IP using Python libs like os or socket

Reply:
Moved it to constants.py file and using localhost ip instead of 10.15.20.2